ocaml/src/proxy_server.ml

```diff
@@ -596,6 +607,29 @@ let proxy_protocol (alba_client : Alba_client.alba_client)
        fun stats () -> alba_client # get_alba_id
     | HasLocalFragmentCache ->
        fun stats () -> alba_client # has_local_fragment_cache |> Lwt.return
+    | UpdateSession ->
+       fun stats args ->
+       begin
+         let result =
+           List.fold_left
+             (fun acc (k,vo) ->
+               match k with
+               | "manifest_ser" ->
+                  begin
+                    match vo with
+                    | None -> acc
+                    | Some v ->
+                       let wanted_version = deserialize Llio.int8_from v in
+                       let new_version = min 2 wanted_version in
+                       let () = ProxySession.set_manifest_ser session new_version in
+                       (k, v) :: acc
```

Comment:
server side should return the actual new_version that it will use to serialize manifests? (and that can be used by the client)